### PR TITLE
fixed the IDMA donor sprites

### PR DESCRIPTION
**Pull request by projectkepler-ru**
## About The Pull Request
This fixes the digitigrade sprites, the file pathing of the idma donor item and add waterpig to the whitelisted item

### CODE REVIEW

File: modular_zubbers/modules/modular_items/code/idmaco_donator.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 /datum/loadout_item/suit/idmarsuit
 	name = "IDMA service jacket"
 	item_path = /obj/item/clothing/suit/hooded/wintercoat/security/idma_jacket
-	ckeywhitelist = list("especiallystrange",  "snaffle15")
+	ckeywhitelist = list("especiallystrange", "waterpig",  "snaffle15")
```
Comment (Cyprex) [inline modular_zubbers/modules/modular_items/code/idmaco_donator.dm:11]: You should probably make a define- `#define IDMA_CKEY_WHITELIST list("[...`
That way you'd only have to edit it in one place

File: modular_zubbers/modules/modular_items/code/idmaco_donator.dm [hunk 2]
```diff
@@ -19,26 +19,30 @@
 
 /datum/loadout_item/under/miscellaneous/idmasnowfatigue
 	name = "IDMA service uniform"
-	item_path = /obj/item/clothing/under/idma_fatigue
-	ckeywhitelist = list("especiallystrange", "snaffle15")
+	item_path = /obj/item/clothing/under/rank/security/idma_fatigue
+	ckeywhitelist = list("especiallystrange", "waterpig",  "snaffle15")
+	restricted_roles = list(JOB_BLUESHIELD, JOB_CAPTAIN, JOB_NT_REP, JOB_HEAD_OF_SECURITY, JOB_RESEARCH_DIRECTOR, JOB_CHIEF_MEDICAL_OFFICER, JOB_CHIEF_ENGINEER, JOB_HEAD_OF_PERSONNEL, JOB_QUARTERMASTER, JOB_SECURITY_OFFICER, JOB_WARDEN, JOB_SECURITY_MEDIC) //Secure Personnel Only. It is a round start armor even if the actual armor can be ordered for cheap
```
Comment (Cyprex) [inline modular_zubbers/modules/modular_items/code/idmaco_donator.dm:24]: if this is a jumpsuit some minor wound bonus is probably appropriate- all jumpsuits have minor wound defense

File: modular_zubbers/modules/modular_items/code/idmaco_donator.dm (hunk 1 above)
Comment (projectkepler-ru) [reply modular_zubbers/modules/modular_items/code/idmaco_donator.dm:11]: Thanks <3

**Review by Majkl-J — COMMENTED** [review]
Changelog would be nice

**Comment by projectkepler-ru** [comment]
> Changelog would be nice

Not necessary

File: modular_zubbers/modules/modular_items/code/idmaco_donator.dm (hunk 2 above)
Comment (Cyprex) [reply modular_zubbers/modules/modular_items/code/idmaco_donator.dm:24]: oh, right, this is the loadout entry... not the actual item- my bad!

File: modular_zubbers/modules/modular_items/code/idmaco_donator.dm [hunk 3]
```diff
@@ -53,23 +57,25 @@
...
+	can_adjust = FALSE //Yes we have the sprite for them but they look ass so let's not use it
 
-/obj/item/clothing/under/idma_fatigue/alt
+/obj/item/clothing/under/rank/security/idma_fatigue/alt
 	name = "argnostan service fatigue"
 	desc = "An old desert uniform worn by the romulus expeditionary force up until march 21st 2181. Attached to it is a silver medal"
 	icon_state = "argnostanuniform"
-	icon = 'modular_zubbers/icons/donator/idmaco.dmi'
-	worn_icon = 'modular_zubbers/icons/mob/clothing/under/idmaco_worn.dmi'
+
```
Comment (Cyprex) [inline modular_zubbers/modules/modular_items/code/idmaco_donator.dm:74]: You can remove this newline too

**Comment by Cyprex** [comment]
Pretty sure the CI failure is unrelated

**Pull request merged**